<launch>
    <arg name="pointcloud_mode" default = "3" />
    <!-- <arg name="camera_odom_name" default = "/odom" /> -->
    <arg name="camera_odom_name" default = "/veo_estimator/camera_pose" />
    <arg name="show_img" default = "false" />
    <arg name="visualize" default="false"/>


    <node name="rgbd2pointcloud" pkg="rgbd2pointcloud" type="rgbd2pointcloud_from_topic" output="screen">
        <param name="pointcloud_mode" type="int" value = "$(arg pointcloud_mode)" />
        <param name="camera_odom_name" type="string" value="$(arg camera_odom_name)"/>
        <param name="show_img" type="bool" value="$(arg show_img)"/>
    </node>

    <node if="$(arg visualize)" name="path_visualisation" pkg="rviz" type="rviz" output="log" args="-d $(find rgbd2pointcloud)/config/pointcloud_visualisation.rviz" />
</launch>


